exercises/practice/two-fer/test/tests.chpl

```diff
@@ -0,0 +1,17 @@
+use UnitTest;
+
+use helloWorld;
+
+proc test_two_fer(test : borrowed Test) throws {
+  test.assertEqual(two_fer(nil), "One for you, one for me.");
```

Comment:
```suggestion
  test.assertEqual(two_fer(), "One for you, one for me.");
```

Thanks, the general test-suite structure looks good, just left a minor check.

Above it says `use helloWorld;`, is this because of manual copy-paste?

Automatically generating testsuites would be very cool, let me know if I can help with specifying chapel grammar.

Reply:
Oh! Yes, I forgot to put a placeholder in my template. Let me fix it!

Reply:
Okay, that should be sorted.

I'll generate up a couple more. It's always easier to figure out what's wrong when looking at specific examples.